guide/ops/locations/index.md

```diff
@@ -124,8 +129,9 @@ For more keys and more detail on the keys below, see
 - Specific Security Groups can be specified using `securityGroups`, as a list of strings (the existing security group names),
   or `inboundPorts` can be set, as a list of numeric ports (selected clouds only)
 
-- A specific existing key pair known at the cloud to use can be specified with `keyPair`
-  (selected clouds only)
+- A specific existing key pair known at the cloud to use for login can be specified with `keyPair`
+  (selected clouds only); if this `keyPair` does not correspond to your default `~/.ssh/id_rsa`
+  then you must typically also specify the `loginUser.privateKeyFile` to use with this `keyPair`
```

Comment:
An existing key pair, configured for login with the cloud can be specified with `keyPair` (selected clouds only). If this `keyPair` is not stored in your default `~/.ssh/id_rsa`, then you must typically also specify the `loginUser.privateKeyFile` which does contain this `keyPair`